example/client/src/di.config.ts

```diff
@@ -16,15 +16,23 @@
 import "../css/diagram.css";
 
 import { createWorkflowDiagramContainer } from "@eclipse-glsp-examples/workflow-sprotty/lib";
-import { GLSPDiagramServer } from "@eclipse-glsp/client";
-import { Container } from "inversify";
+import { configureActionHandler, GLSPDiagramServer } from "@eclipse-glsp/client";
+import { Container, ContainerModule } from "inversify";
 import { ConsoleLogger, LogLevel, TYPES } from "sprotty";
 
+import { KeepAliveAction } from "./keep-alive-action";
+import { KeepAliveActionHandler } from "./keep-alive-action-handler";
 
 export default function createContainer(): Container {
     const container = createWorkflowDiagramContainer('sprotty');
     container.bind(TYPES.ModelSource).to(GLSPDiagramServer).inSingletonScope();
     container.rebind(TYPES.ILogger).to(ConsoleLogger).inSingletonScope();
     container.rebind(TYPES.LogLevel).toConstantValue(LogLevel.warn);
```

Comment:
Maybe we should use `LogLevel.info` here? Otherwise the message you log in 'KeepAliveActionHandler' is never printed to the console.

Reply:
I guess that usually you don't want to see the "Keep Alive Action received" logs, that's why I chose a lower log level than the default. I just used it to debug whether the actions are received by the client at all. 
Setting the default to info might spam the log with other messages as well.

Reply:
:+1: agreed